.github/workflows/test_book.yml

```diff
@@ -0,0 +1,40 @@
+name: Testnet Test Book
+
+on:
+  push:
+    branches: ["testbook"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        default: true
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y libgirepository1.0-dev libleveldb-dev
+        python -m pip install --upgrade pip
+        pip install pylint maturin pytest
+        cd counterparty-rs && pip install -e . && cd ..
+        cd counterparty-lib && pip install -e . && cd ..
+        cd counterparty-cli && pip install -e . && cd ..
+    - name: Bootstrap testnet database
+      run: |
+        counterparty-server --backend-password=rpc --testnet bootstrap
```

Comment:
isn't `rpc` the default backend pass? (i.e. do we need to pass it here explicitly?)

Reply:
yep! I restored the default password!